File: Beginner/Bear and Milky Cookies (COOMILK)/cookies.py
t=int(input())
 

while t:
    N = int(input())
    S = list(input().split())
    f = 0

    for j in range(N):
        if (j+1)<N and S[j]=="cookie" and S[j+1]!="milk":
            f=1
            break
        elif S[N-1]=="cookie":
            f=1
            break
        
    if f==1:
        print("NO")
    else:
        print("YES")

    t = t-1

File: Beginner/Bear and Milky Cookies (COOMILK)/cookies.cpp
#include<bits/stdc++.h>
using namespace std;


int main() {
   int t, n, f=0;
   cin>>t;
   while(t--) {
        f = 0;
        cin>>n;
        string s[n];
        for(int i=0;i<n;i++) {
            cin>>s[i];
        }
        for(int i=0;i<n;i++){
            if(s[i]=="cookie" && s[i+1]!="milk"){
            f = 1;
            }
        }
        if(f==0) 
            cout<<"YES"<<endl;
        else 
            cout<<"NO"<<endl;
   }
   return 0;
}